modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -0,0 +1,208 @@
+|The maximum size, in GiB, of each archive file within the image set.
+|Integer. For example: `4`
+
+|`mirror`
+|The configuration of the image set.
+|Object
+
+|`mirror.additionalImages`
+|The additional images configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+----
+
+|`mirror.additionalImages.name`
+|The tag of the image to mirror.
+|String. For example: `registry.redhat.io/ubi8/ubi:latest`
+
+|`mirror.helm`
+|The helm configuration of the image set. Note that the oc-mirror plug-in only supports helm charts that do not require user input when rendered.
+|Object
+
+|`mirror.helm.local`
+|The local helm charts to mirror.
+|Array of objects. For example:
+
+[source,yaml]
+----
+local:
+  - name: podinfo
+    path: /test/podinfo-5.0.0.tar.gz
+----
+
+|`mirror.helm.local.name`
+|The name of the local helm chart to mirror.
+|String. For example: `podinfo`.
+
+|`mirror.helm.local.path`
+|The path of the local helm chart to mirror.
+|String. For example: `/test/podinfo-5.0.0.tar.gz`.
+
+|`mirror.helm.repos`
+|The remote helm repositories to mirror from.
+|Array of objects. For example:
+
+[source,yaml]
+----
+repos:
+  - name: podinfo
+    url: https://example.github.io/podinfo
+    charts:
+      - name: podinfo
+        version: 5.0.0
+----
+
+|`mirror.helm.repos.name`
+|The name of the helm repository to mirror from.
+|String. For example: `podinfo`.
+
+|`mirror.helm.repos.url`
+|The URL of the helm repository to mirror from.
+|String. For example: [x-]`https://example.github.io/podinfo`.
+
+|`mirror.helm.repos.charts`
+|The remote helm charts to mirror.
+|Array of objects.
+
+|`mirror.helm.repos.charts.name`
+|The name of the helm chart to mirror.
+|String. For example: `podinfo`.
+
+|`mirror.helm.repos.charts.version`
+|The version of the named helm chart to mirror.
+|String. For example: `5.0.0`.
+
+|`mirror.ocp`
+|The platform configuration of the image set.
+|Object
+
+|`mirror.ocp.channels`
+|The platform channel configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+channels:
+  - name: stable-4.7
+  - name: stable-4.6
+    versions:
+      - '4.6.36'
+----
+
+|`mirror.ocp.channels.name`
+|The name of the release channel.
+|String. For example: `stable-4.9` or `okd`.
+
+|`mirror.ocp.channels.versions`
+|The list of release versions within the named channel.
+|String. For example: `4.9.6`.
+
+|`mirror.operators`
+|The Operators configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9
+    headsOnly: false
+    packages:
+      - name: elasticsearch-operator
+        startingVersion: '2.4.0'
+----
+
+|`mirror.operators.catalog`
+|The Operator catalog to include in the image set.
+|String. For example: `registry.redhat.io/redhat/redhat-operator-index:v4.9`.
+
+|`mirror.operators.headsOnly`
```

Comment:
This boolean cannot be used if the packages subkey is present. Can you please add a note about that here?